src/components/Search/Search.js

```diff
@@ -7,29 +7,30 @@ import ErrorMessage from '../Errors/ErrorMessage/ErrorMessage'
 import { getProperties } from '../../utils/frontend-api-client/properties'
 
 const Search = ({ query }) => {
-  const [searchParams, setSearchParams] = useState('')
+  const [searchQuery, setSearchQuery] = useState('')
   const [properties, setProperties] = useState([])
   const [loading, setLoading] = useState(false)
-  const [searching, setSearching] = useState(true)
-  const [searchQuery, setSearchQuery] = useState('')
   const [error, setError] = useState()
   const router = useRouter()
+  const patt = /[0-9]{8}/g
```

Comment:
Could we rename this to something a bit more descriptive? 
i.e.`workOrderReferencePattern` or I think Neil may have mentioned `workOrderReferenceRegex`

Reply:
As a foreign user to the system I'm not sure they would know what this meant at initial glance